.github/workflows/stale.yaml

```diff
@@ -0,0 +1,42 @@
+name: 'Close stale issues and PRs'
+
+on:
+  push:
+    paths:
+      - .github/workflows/stale.yml
+    branches-ignore:
+      - dependabot/**
```

Comment:
We should also ignore release branches like `0.6.x`.

Going forward we could always have a release branch with fixed prefix `release-`.

Reply:
Will add the `release-*` regex for ignore list.

Is there active development still ongoing for `0.6.x` branch considering there's a `1.x.x` release? 🤔 

If not, I believe that branch can be deleted. Any `0.6.x` tags won't be deleted.

Reply:
No active development on `0.6.x` branch, and I don't see a problem with deleting it. 

For being on the safe side, let me create a `release-0.6.x` branch from `0.6.x`.

Reply:
Can you please create the branch as `release/0.6.x`?

There are some automation, for example dependabot, which creates branches with the "namespace" and "name" separated by `/` instead of `-`, e.g. - `dependabot/github_actions/helm/kind-action-1.8.0`